.github/workflows/deployBlocker.yml

```diff
@@ -0,0 +1,50 @@
+name: Update Deploy Blockers
+
+on:
+  pull_request:
+    types:
+      - labeled
+  issues:
+    types:
+      - labeled
+
+jobs:
+  deployBlocker:
+    runs-on: ubuntu-latest
+    if: ${{ github.event.label.name == 'DeployBlockerCash' }}
+
+    steps:
+      - name: Update StagingDeployCash with issue
+        uses: Expensify/Expensify.cash/.github/actions/createOrUpdateStagingDeploy@master
+        if: ${{ github.event_name == 'issues' }}
+        with:
+          GITHUB_TOKEN: ${{ secrets.OS_BOTIFY_TOKEN }}
+          NEW_PULL_REQUESTS: ${{ github.event.issue.url }}
```

Comment:
ooo I think you might need to use the `html_url` here so it doesn't use the API base URL

Reply:
Ah good catch, agree

Reply:
See the difference:

![image](https://user-images.githubusercontent.com/47436092/111012661-a47a4c00-8351-11eb-8b33-fcc10817f0ea.png)